README.md

```diff
@@ -0,0 +1,38 @@
+# Some Engineering Helm Chart Repository
+
+![](https://user-images.githubusercontent.com/2124094/164599444-448f92f1-7a73-4a86-a377-bc650f10e756.png)
```

Comment:
Can we add placeholder test here `RESOTO` or something like that?

Reply:
This is a decorative image and thus should not have alt text.